.github/workflows/check_pr_title.yml

```diff
@@ -1,7 +1,14 @@
-name: Check commit messages
-on: push
+name: PR title
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+
 jobs:
-  check-commit:
+  check:
```

Comment:
Please change the script file name to the new name in the `run` command, the workflow fails at the moment.